lib/active_fedora/reflection.rb

```diff
@@ -477,5 +585,66 @@ def build_index_config
           ActiveFedora::Indexing::Map::IndexObject.new(predicate_for_solr) { |index| index.as :symbol }
         end
     end
+
+    class SingularRDFPropertyReflection < RDFPropertyReflection
+      def macro
+        :singular_rdf
+      end
+
+      def collection?
+        false
+      end
+
+      def association_class
+        Associations::SingularRDF
+      end
+    end
+
+    class FilterReflection < ActiveFedora::Reflection::AssociationReflection
```

Comment:
Does this (and line 624) need the explict `ActiveFedora::Reflection` namespacing (e.g., because without it an AR constant is resolved)? I see bare `AssociationReflection`s in other places in this file.